styles/base.css

```diff
@@ -0,0 +1,45 @@
+/* --------------------selectors ------------------------*/
+
+body {
+  background-color: blanchedalmond;
+  font-family: poppins, sans-serif;
+  font-weight: light 300;
+}
+
+h1 {
+  font-weight: 500;
+  letter-spacing: 0.25em;
+  display: flex;
+  justify-content: center;
```

Comment:
display: flex;
  justify-content: center;

if you want to center text `text-align: center;` might be a better choice